lib/index.js

```diff
@@ -54,7 +54,7 @@ internals.Connection.prototype.start = function (callback) {
 
     // Wait for connection
 
-    client.once('connect', function () {
```

Comment:
Ah, true! 

I was getting un-ready connections later in the tests. I suppose enough of that interface is exposed that that could be set, eh?